python/pyraphtory/pyraphtory/debug.py

```diff
@@ -0,0 +1,19 @@
+try:
```

Comment:
do you still need this?

Reply:
Something like this is necessary to get the debugger to work for the callbacks. The problem is that they run in threads started from the jvm which are otherwise ignored.